### PR TITLE
Add System.Net.IPAddress missing members

### LINKED ISSUE #12131: Port System.Net.IPAddress missing members
```csharp
    public class IPAddress {
        public long Address { get; set; }
    }
```

### PR DESCRIPTION
**Pull request by tijoytom**
Add back Address property

Fixes #12131

### CODE REVIEW

**Comment by tijoytom** [comment]
@karelz @davidsh @ericeil PTAL

File: src/System.Net.Primitives/src/System/Net/IPAddress.cs [hunk 1]
```diff
@@ -150,6 +150,42 @@ public IPAddress(byte[] address, long scopeid)
             PrivateScopeId = (uint)scopeid;
         }
 
+        public long Address
```
Comment (justinvp) [inline src/System.Net.Primitives/src/System/Net/IPAddress.cs]: Shouldn't it be `[Obsolete]`?

File: src/System.Net.Primitives/tests/FunctionalTests/IPAddressTest.cs [hunk 2]
```diff
@@ -251,6 +251,16 @@ public static void IsIPV6Teredo_Get_Success()
...
         [Fact]
+        public static void Address_Property_Success()
+        {
+            IPAddress ip1 = IPAddress.Parse("192.168.0.9");
+            //192.168.0.10
+            long newIp4Address = 192 << 24 | 168 << 16 | 0 << 8 | 10;
+            ip1.Address = newIp4Address;
+            Assert.Equal(ip1.ToString(), "192.168.0.10");
+        }
+
```
Comment (stephentoub) [inline src/System.Net.Primitives/tests/FunctionalTests/IPAddressTest.cs]: Test for IPv6 throwing behavior?

File: src/System.Net.Primitives/ref/System.Net.Primitives.cs [hunk 3]
```diff
@@ -205,6 +205,7 @@ public partial class IPAddress
         public static System.Net.IPAddress Parse(string ipString) { return default(System.Net.IPAddress); }
         public override string ToString() { return default(string); }
         public static bool TryParse(string ipString, out System.Net.IPAddress address) { address = default(System.Net.IPAddress); return default(bool); }
+        public long Address { get; set; }
```
Comment (tijoytom) [inline src/System.Net.Primitives/ref/System.Net.Primitives.cs]: Not sure about that , this's the desktop message " [ObsoleteAttribute("This property has been deprecated. It is address family dependent. Please use IPAddress.Equals method to perform comparisons. http://go.microsoft.com/fwlink/?linkid=14202")]" . BTW , the link is broken and i am not sure it talks about comparison when it's about setting and getting the Address property

File: src/System.Net.Primitives/tests/FunctionalTests/IPAddressTest.cs (hunk 2 above)
Comment (tijoytom) [inline src/System.Net.Primitives/tests/FunctionalTests/IPAddressTest.cs]: Done

File: src/System.Net.Primitives/src/System/Net/IPAddress.cs [hunk 4]
```diff
@@ -150,6 +150,42 @@ public IPAddress(byte[] address, long scopeid)
...
+                // IPv6 Changes: Can't do this for IPv6, so throw an exception.
+                //
+                //
+                if (AddressFamily == AddressFamily.InterNetworkV6)
+                {
+                    throw new SocketException(SocketError.OperationNotSupported);
+                }
+                else
+                {
+                    return (long)PrivateAddress;
```
Comment (tijoytom) [inline src/System.Net.Primitives/src/System/Net/IPAddress.cs]: Removed

Comment (tijoytom) [inline src/System.Net.Primitives/src/System/Net/IPAddress.cs]: removed

File: src/System.Net.Primitives/ref/System.Net.Primitives.cs (hunk 3 above)
Comment (davidsh) [inline src/System.Net.Primitives/ref/System.Net.Primitives.cs]: You need to put the `[Obsolete]` attributes into the ref just like Desktop

**Comment by stephentoub** [comment]
Replaced by https://github.com/dotnet/corefx/pull/12658

**Pull request closed without merging**